main/src/main/java/io/github/chains_project/cs/Main.java

```diff
@@ -210,10 +235,16 @@ public FileVisitResult postVisitDirectory(Path dir, IOException e) throws IOExce
     private static InvocationResult mavenTestInvoker(Revision project) throws IOException, MavenInvocationException {
         InvocationRequest request = new DefaultInvocationRequest();
         request.setPomFile(project.resolveFilename("pom.xml").toFile());
+        try {
+            project.resolveFilename(".mvn/maven.config").toFile().delete();
+        } catch (FileNotFoundException e) {
+            // ignore
+        }
```

Comment:
Could you tell me what is in this configuration file? I would say we remove environment dependent factors from code because it makes it difficult for the code to be generic and work on different systems.

Reply:
We add it because https://github.com/schemacrawler/SchemaCrawler/tree/07c73685700ce71e78719b7c2249dfb578bd4b84  could not be run with maven `3.9.1` as there is a [breaking change ](https://stackoverflow.com/questions/75469591/maven-build-failing-due-to-unrecognized-settings-option)in how the configuration file should be specified.